src/interfaces.ts

```diff
@@ -9,6 +9,12 @@ export interface ISigningResponse {
   signatures: string[];
 }
 
+interface ISideChainInfo {
```

Comment:
Is this side chain or chain account info? I think this object is maybe not named well? Maybe it should just be `AccountInfo`